vpn.go

```diff
@@ -0,0 +1,46 @@
+package packngo
+
+import "fmt"
+
+const vpnBasePath = "/user/vpn"
+
+// VPNConfig struct
+type VPNConfig struct {
+	Config string `json:"config,omitempty"`
```

Comment:
maybe remove the `omitempty`? If the openvpn config is not in the response, the fucntion call should fail.

Reply:
Let's leave it. It has no effect here. Whatever you do,  you will always check for errors so you won't be accessing `Config`.